temporal/api/cloud/namespace/v1/message.proto

```diff
@@ -110,10 +114,9 @@ message Namespace {
     Endpoints endpoints = 6;
     // The limits set on the namespace currently.
     Limits limits = 7;
-    // The private link for the namespace, if any.
-    oneof private_link {
-    	AWSPrivateLink aws_private_link = 8;
-    }
+    // The private connectivity for the namespace by region where they apply, if any.
+    // The region is the key and the private connectivity is the value.
+    map<string, PrivateConnectivity> private_connectivity = 8;
```

Comment:
Is every region-based configuration going to have to be a `map<string, ...`? Maybe region is an object with information about itself.